### PR TITLE
[python] Add get_obs

### PR DESCRIPTION
**Pull request by ivirshup**
Fixes https://github.com/chanzuckerberg/cellxgene-census/issues/1037

### TODO:

- [x] Add `get_var`
- [x] Docs
- [x] Confirm argument names

### CODE REVIEW

**Review by prathapsridharan — CHANGES_REQUESTED** [review]
Overall this looks good to me. I have one question about the implementation in the test case.

@pablo-gar @ebezzi - Can y'all take a look too please?

File: api/python/cellxgene_census/tests/test_get_anndata.py
Comment (ivirshup) [reply api/python/cellxgene_census/tests/test_get_anndata.py:309]: Yes, I'll update the comment to be more clear.

**Review by prathapsridharan — APPROVED** [review]
LGTM but please wait for @ebezzi and @pablo-gar to review as well before merging to `main`

**Review by pablo-gar — APPROVED** [review]
@prathapsridharan @ebezzi I see a pattern in the file structure in `api/python/cellxgene_census/src/cellxgene_census` whereby the SOMAExperiment helpers are under `_experiment.py`, the anndata helpers are `_get_anndata.py`

Would it make sense to have the proposed helpers in this PR under a new `_axis_metadata.py` or `_get_axis_metadata.py` file?   

I defer to you both as this touches dev eng design.

LGTM otherwise

**Comment by prathapsridharan** [comment]
`get_obs` and `get_var` to me seems consistent when put together with `anndata` related functions. And the same applies to helper functions needed in service of `get_anndata`, `get_obs`, `get_var`.

I think we can shift things around later if needed but for now this organization seems fine to me

**Review by ebezzi — CHANGES_REQUESTED** [review]
Typo in the docs, but otherwise LGTM.

**Review by ebezzi — APPROVED** [review]

**Pull request merged**
